### ISSUE TITLE
module 'ipyparallel' has no attribute 'Cluster'

### ISSUE THREAD

**Issue by S-Yuan137** [ipython/ipyparallel#654]
Hi, I am new to parallel programming. When I tried to use ipyparallel, I met a strange error:

![image](https://user-images.githubusercontent.com/73228164/147647757-a105fa43-4362-4dce-8023-982dbba0f1bf.png)

As shown in the figure, it has no `ipyparallel.Cluster()` attribute but is fine with `Client()`, which is unreasonable. Here I use VScode to run JupyterNoteBook on a remote Linux cluster. My python version is 3.8.12, and other environment configurations are shown below:
```
(py38) me@remote_cluster:~/PythonCodes$ jupyter --version
Selected Jupyter core packages...
IPython          : 7.29.0
ipykernel        : 6.4.1
ipywidgets       : not installed
jupyter_client   : 7.0.6
jupyter_core     : 4.9.1
jupyter_server   : not installed
jupyterlab       : not installed
nbclient         : 0.5.9
nbconvert        : 6.3.0
nbformat         : 5.1.3
notebook         : 6.4.6
qtconsole        : not installed
traitlets        : 5.1.1
```
And the ipyparallel version is 8.1.0.
I wonder why I can't use `ipyparallel.Cluster()`.  Thanks for any advice and help.

**Comment by minrk** [comment]
What's `ipp.__version__`? It should be at least 7.

**Comment by S-Yuan137** [comment]
> What's `ipp.__version__`? It should be at least 7.

Oh, you remind me!

The weird thing is that the error disappeared now. I presume that the error is from the residual environmental configuration because I updated the package from 6.2.0 to 8.1.0. At that time I only checked the version in the terminal instead of in the code snippets cell.   

Now I check it in the code cell with `ipp.__version__` and it gives `8.1.0`. Everything seems fine now.

Thank you!
